web/packages/teleterm/src/services/config/appConfigSchema.ts

```diff
@@ -114,6 +114,10 @@ export const createAppConfigSchema = (platform: Platform) => {
       .boolean()
       .default(false)
       .describe('Enables sharing the computer.'),
+    'feature.headlessSkipConfirm': z
```

Comment:
```suggestion
    'headless.skipConfirm': z
```

I know this is not well explained anywhere, but typically we use the `feature` prefix for feature flags that are eventually going to be removed.